uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-calendar-month.component.spec.js

```diff
@@ -473,9 +473,9 @@
             });
           });
 
-          describe('when the balance change of the leave request is positive', function () {
```

Comment:
what happens to other request types with a positive balance? are we deleting that functionality?